genesis-config/src/lib.rs

```diff
@@ -217,6 +216,17 @@ impl GenesisConfig {
     }
 }
 
+#[cfg(feature = "serde")]
+fn lamports_to_sol_str(lamports: u64) -> String {
```

Comment:
imo `GenesisConfig` implementing `Display` is a bug. how about we just switch the output to lamports?

Reply:
Fine by me, done in [c4b0eef](https://github.com/anza-xyz/solana-sdk/pull/174/commits/c4b0eef468028ea3f5d7f5a3e9a916d5659cdce1)